### DISCUSSION TITLE
Fail to compile the codes

### DISCUSSION THREAD

**Discussion by Tu-Shiqin** [Anura3D/Anura3D_OpenSource#89] [Q&A] [answered]
I have followed the manual, and installed the VS Community version 2019 (16.11.25) and the oneAPI package (Base + HPC Toolkits). But I failed to compile the codes. The error in the output of VS is shown below. Could you please tell me the potential causes? Thank you very much.
![image](https://user-images.githubusercontent.com/125841317/226780285-d0309167-a974-4fe1-a5ea-6c880376a3d0.png)
![image](https://user-images.githubusercontent.com/125841317/226780292-d0033e77-2541-450d-92da-07e88c31a332.png)
![image](https://user-images.githubusercontent.com/125841317/226780348-0cfdb0fb-2ff1-4866-a35b-3538be918395.png)

**Comment by ayerrocolom** [comment, marked as answer]
Dear Tu-Shiquin,
Unfortunately, I don't know what is causing the error in your code based on your screenshots. I would encourage you to use the debugging tool in Visual Studio to see what is not working for you.
Best,
Alba
